File: programmers/lv1/복습필요/숫자문자열과영단어.py
def solution(s):
    arr = ['zero', 'one', 'two', 'three', 'four','five','six','seven','eight','nine']
    a = ''
    for num, i in enumerate(arr):
        
        if i in s:
            s= s.replace(i, str(num))
        a = s    
    return int(a)

File: src/programmers/숫자문자열과영단어.java
class Solution {
    public int solution(String s) {
        int answer = 0;
        String[] stringArray = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};
        int[] numberArray = {0,1,2,3,4,5,6,7,8,9};

        
        for(int i=0; i<stringArray.length;i++){
            
         s = s.replace(stringArray[i], Integer.toString(numberArray[i]));
            
        }
        
        
        return Integer.parseInt(s);
    }
}
